pyanaconda/network.py

```diff
@@ -1524,6 +1529,49 @@ def _update_vlan_interfacename_ksdata(devname, ndata):
     if devname != default_ks_vlan_interface_name(ndata.device, ndata.vlanid):
         ndata.interfacename = devname
 
+def update_slaves_onboot_value(devname, value):
+    """Update onboot value in ifcfg files of device slaves
+
+    :param devname: name of device
+    :type devname: str
+    :param value: value of onboot setting
+    :type value: bool
+    :returns: list of names of updated connections
+    :rtype: list of strings
+
+    """
+    retval = []
+    if value:
+        ifcfg_value = 'yes'
+    else:
+        ifcfg_value = 'no'
+
+    # Master can be identified by devname or uuid, find master uuid
+    try:
+        uuid = nm.nm_device_setting_value(devname, "connection", "uuid")
+    except nm.UnknownDeviceError:
+        # Until activated, the device does not exist, so look in its ifcfg file
+        ifcfg_path = find_ifcfg_file_of_device(devname)
+        if not ifcfg_path:
+            log.debug("network: can't find ifcfg file of %s", devname)
+            return retval
+        ifcfg = IfcfgFile(ifcfg_path)
+        ifcfg.read()
+        uuid = ifcfg.get('UUID')
+
+    # Find and update ifcfg files of slaves
+    for filepath in _ifcfg_files(netscriptsDir):
```

Comment:
Where does ``netscriptsDir`` come from here? Is it a global constant in the file?